web-marketplace/src/lib/auth/auth.tsx

```diff
@@ -0,0 +1,73 @@
+import { createContext, useContext } from 'react';
```

Comment:
important file to review, this will need to be updated to get the private account fields once regen-network/regen-server#407 is done